### PR TITLE
Fix ReadMe to be in markdown and organized it a little bit

### PR DESCRIPTION
**Pull request by rluvaton**
Usually, the README file is in markdown syntax

Markdown is really popular these days, StackOverflow, GitHub, And much more use Markdown

## Markdown Introduction
Markdown is a way to style text on the web. You control the display of the document; formatting words as bold or italic, adding images, and creating lists are just a few of the things we can do with Markdown. Mostly, Markdown is just regular text with a few non-alphabetic characters thrown in, like `#` or `*`.

## Useful links
- Learn more about Markdown [here (_Wikipedia_)](https://en.wikipedia.org/wiki/Markdown)
- Mastering Markdown [here (_GitHub Guides_)](https://guides.github.com/features/mastering-markdown/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Python Session
 > Author: Daniel Huri
 > 
-> This was Built in Python 3.6+
```
Comment (hurihuri123) [inline README.md]: It was actually written using python 2.7

Comment (rluvaton) [reply README.md]: Yeah, my bad, I saw prints with parenthesis (`(`, `)`) so I assumed it was python 3.6

Comment (rluvaton) [reply README.md]: It fixed in the second commit...

Comment (rluvaton) [reply README.md]: Maybe I needed to [amend](
https://www.atlassian.com/git/tutorials/rewriting-history) both commits

**Pull request merged**